lib/base/strings.dart

```diff
@@ -282,7 +282,7 @@ abstract class Strings {
   static const analogIOBody =
       "AnalogIO is a group of volunteer software students, supporting the student organization Cafe Analog.\n\nWe create and maintain backend systems, apps and websites for both customers and baristas of the cafe.\n\nWant to help? Have any feedback? Don't hesitate to contact us.";
   static const github = 'GitHub';
-  static const sendUsAnEmail = 'Send us an email';
+  static const provideFeedback = 'Provide feedback';
```

Comment:
How about calling it "Report a bug or give feedback"?

Reply:
Done